checkbox-ng/plainbox/impl/unit/testplan.py

```diff
@@ -843,6 +843,8 @@ def visit_IncludeStmt_node(self, node: IncludeStmt):
                     override_list.append((pattern, field_value_list))
 
             V().visit(IncludeStmtList.parse(testplan.include))
+            if testplan.mandatory_include:
+                V().visit(IncludeStmtList.parse(testplan.mandatory_include))
```

Comment:
this is below `if testplan.include is not None`. Is this intended? Can a test have no include field and mandatory include? Also, what about `boostrap_include`?